.github/buildomat/jobs/package.sh

```diff
@@ -52,88 +28,33 @@ ptime -m cargo run --locked --release --bin omicron-package -- \
   -t test target create -i standard -m non-gimlet -s softnpu -r single-sled
 ptime -m cargo run --locked --release --bin omicron-package -- \
   -t test package
+mapfile -t packages \
+  < <(cargo run --locked --release --bin omicron-package -- -t test list-outputs)
 
 # Build the xtask binary used by the deploy job
 ptime -m cargo build --locked --release -p xtask
 
-# Assemble some utilities into a tarball that can be used by deployment
-# phases of buildomat.
+# Build the end-to-end tests
+# Reduce debuginfo just to line tables.
+export CARGO_PROFILE_DEV_DEBUG=1
+export CARGO_PROFILE_TEST_DEBUG=1
```

Comment:
https://doc.rust-lang.org/cargo/reference/profiles.html#debug suggests that `1` is more than just line-tables-only.

Can we use the full string here?

Reply:
I think this changed since I first wrote this line (this was moved from ci-tools.sh). I'll investigate.